scene/resources/font.cpp

```diff
@@ -1526,798 +1522,846 @@ void FontData::remove_cache(int p_cache_index) {
+
+	ClassDB::bind_method(D_METHOD("draw_string_outline", "canvas_item", "pos", "text", "alignment", "width", "size", "modulate", "flags", "direction", "orientation"), &FontConfig::draw_string_outline, DEFVAL(HORIZONTAL_ALIGNMENT_LEFT), DEFVAL(-1), DEFVAL(1), DEFVAL(Color(1.0, 1.0, 1.0)), DEFVAL(TextServer::JUSTIFICATION_KASHIDA | TextServer::JUSTIFICATION_WORD_BOUND), DEFVAL(TextServer::DIRECTION_AUTO), DEFVAL(TextServer::ORIENTATION_HORIZONTAL));
+	ClassDB::bind_method(D_METHOD("draw_multiline_string_outline", "canvas_item", "pos", "text", "alignment", "width", "max_lines", "size", "modulate", "flags", "direction", "orientation"), &FontConfig::draw_multiline_string_outline, DEFVAL(HORIZONTAL_ALIGNMENT_LEFT), DEFVAL(-1), DEFVAL(-1), DEFVAL(1), DEFVAL(Color(1.0, 1.0, 1.0)), DEFVAL(TextServer::BREAK_MANDATORY | TextServer::BREAK_WORD_BOUND | TextServer::JUSTIFICATION_KASHIDA | TextServer::JUSTIFICATION_WORD_BOUND), DEFVAL(TextServer::DIRECTION_AUTO), DEFVAL(TextServer::ORIENTATION_HORIZONTAL));
+
+	ClassDB::bind_method(D_METHOD("has_char", "char"), &FontConfig::has_char);
+	ClassDB::bind_method(D_METHOD("get_supported_chars"), &FontConfig::get_supported_chars);
+
+	ClassDB::bind_method(D_METHOD("get_char_size", "char"), &FontConfig::get_char_size);
+	ClassDB::bind_method(D_METHOD("draw_char", "canvas_item", "pos", "char", "modulate"), &FontConfig::draw_char, DEFVAL(Color(1.0, 1.0, 1.0)));
+	ClassDB::bind_method(D_METHOD("draw_char_outline", "canvas_item", "pos", "char", "size", "modulate"), &FontConfig::draw_char_outline, DEFVAL(-1), DEFVAL(Color(1.0, 1.0, 1.0)));
+
+	ADD_PROPERTY(PropertyInfo(Variant::OBJECT, "font", PROPERTY_HINT_RESOURCE_TYPE, "Font"), "set_font", "get_font");
 
-	ClassDB::bind_method(D_METHOD("update_changes"), &Font::update_changes);
+	ADD_PROPERTY(PropertyInfo(Variant::INT, "size", PROPERTY_HINT_RANGE, "1,4000,1"), "set_size", "get_size");
 
-	ClassDB::bind_method(D_METHOD("get_rids"), &Font::get_rids);
+	ADD_GROUP("Variation", "variation");
+	ADD_PROPERTY(PropertyInfo(Variant::DICTIONARY, "variation_opentype"), "set_variation_opentype", "get_variation_opentype");
+	ADD_PROPERTY(PropertyInfo(Variant::FLOAT, "variation_embolden", PROPERTY_HINT_RANGE, "-2,2,0.01"), "set_variation_embolden", "get_variation_embolden");
+	ADD_PROPERTY(PropertyInfo(Variant::TRANSFORM2D, "variation_transform"), "set_variation_transform", "get_variation_transform");
+	ADD_PROPERTY(PropertyInfo(Variant::BOOL, "variation_customize_fallbacks"), "set_variation_customize_fallbacks", "get_variation_customize_fallbacks");
+
+	ADD_GROUP("OpenType features", "opentype");
```

Comment:
```suggestion
	ADD_GROUP("OpenType Features", "opentype");
```
Just something small I noticed when looking at the screenshots.